src/presentation/pages/NFT/NFTPage.tsx

```diff
@@ -0,0 +1,16 @@
+import { TopBarLayout } from 'infrastructure/components/Layout/TopBarLayout';
+import { Main } from './Main/Main';
+
+const NFTPage = () => (
+  <>
+    <TopBarLayout
+      pageComponent={
+        <>
+          <Main />
+        </>
+      }
+    />
+  </>
+);
+
```

Comment:
This fragment seems to be unneccesary 
```suggestion

const NFTPage = () => (
    <TopBarLayout
      pageComponent={<Main />}
    />
);

```